packages/sdk-js/src/feature-repository.ts

```diff
@@ -299,28 +299,42 @@ function startAutoRefresh(instance: GrowthBook) {
               clientKey,
               error: e ? (e as Error).message : null,
             });
-          onSSEError(channel, key);
+          onSSEError(instance, channel, key);
```

Comment:
I don't like the idea of passing `instance` here.  The SSE connection should be independent of any instances.  In Node, for example, a single SSE connection might be attached to 100 separate GrowthBook instances.

Instead of destroying the channel and calling `startAutoRefresh` again, might be better to keep the channel around, close the `src`, and mark it as inactive or something.  When we retry, we just need to create a new `src`. 
 That way, all the instance references remain intact and we are basically just refreshing the connection transparently.